wordpress-mobile/WordPress-Android
# Onboarding Improvements Existing Users: Tracking

Fixes #15010 

This PR adds tracking for the OnBoarding Existing Users project

Two new events were added:
LOGIN_EPILOGUE_CHOOSE_SITE_TAPPED 
LOGIN_EPILOGUE_CREATE_NEW_SITE_TAPPED

Also added `CHOOSE_SITE` to UnifiedLoginTracker.Click  

To test:
- Launch the app 
- Enable `My Site` -> `App Settings` -> 'Privacy Settings' ->  "Collecting Information"   
- Go to `My Site` -> `App Settings` -> `Test feature configuration`.
- Enable the `OnboardingImprovementsFeatureConfig` and click on the `RESTART THE APP` button 
- Log out of the app
- Log in to the app
- On the Login Epilogue tap the `Create a new site` button
- Check log for:
- `🔵 Tracked: login_epilogue_create_new_site_tapped`
- `🔵  Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_password","step":"success","click":"create_new_site"}`
- Log out of the app

- Log in to the app
- On the Login Epilogue, Tap on a site to select it
- - Check log for:
- `🔵 Tracked: login_epilogue_choose_site_tapped`
- `🔵  Tracked: unified_login_interaction, Properties: {"source":"default","flow":"login_password","step":"success","click":"choose_site"}`

## Regression Notes
1. Potential unintended areas of impact 
The new events aren't tracked

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested and viewed log files

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
